test/fixtures/parser/postgres/postgres_create_dollar_quoted_function.sql

```diff
@@ -0,0 +1,3 @@
+CREATE FUNCTION foo(integer, integer) RETURNS integer
+    AS $$ select $1 + $2; $$
```

Comment:
Does this also cover `$whatever$` including nesting like below:

```SQL
CREATE FUNCTION foo(TEXT, TEXT) RETURNS integer
    AS $func$ select $1 || $2 || $str$text with more dollars in it $ dsfsd $ dsf$str$; $func$
    LANGUAGE SQL;
```

I used this extensively to create dynamic SQL within a function.

Reply:
It should do!